tools/topology/topology2/cavs/platform/intel/alh-2nd-spk.conf

```diff
@@ -0,0 +1,18 @@
+
+Object.Pipeline {
+	passthrough-be.$ALH_2ND_SPK_ID {
```

Comment:
is this really passthrough here?

Reply:
> is this really passthrough here?

Yes, basically, passthrough-be pipeline contains a BE dai copier and that's what we need.